From 3cebbd82d6ccea15d7f8e51097f6318f9b369c34 Mon Sep 17 00:00:00 2001
From: Antonio Borneo <antonio.borneo@foss.st.com>
Date: Mon, 17 Oct 2022 16:16:23 +0200
Subject: [PATCH 070/529] ARM: dts: stm32: add exti entry 44 for USB_OTG on
 stm32mp151

Add exti entry 44 used for USB_OTG wake up.
This event is mapped to GIC 98.

Signed-off-by: Antonio Borneo <antonio.borneo@foss.st.com>
Signed-off-by: Alexandre Torgue <alexandre.torgue@foss.st.com>
Change-Id: I33c2237363e34b309b0d4c1aa4d528909672b6d0
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/274971
Reviewed-by: Antonio Maria BORNEO <antonio.borneo@st.com>
Reviewed-by: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
Tested-by: Antonio Maria BORNEO <antonio.borneo@st.com>
Domain-Review: Amelie DELAUNAY <amelie.delaunay@foss.st.com>
---
 arch/arm/boot/dts/stm32mp151.dtsi | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm/boot/dts/stm32mp151.dtsi
+++ b/arch/arm/boot/dts/stm32mp151.dtsi
@@ -1249,6 +1249,7 @@
 					<32 0 &intc GIC_SPI 82  IRQ_TYPE_LEVEL_HIGH>,
 					<33 0 &intc GIC_SPI 83  IRQ_TYPE_LEVEL_HIGH>,
 					<43 0 &intc GIC_SPI 75  IRQ_TYPE_LEVEL_HIGH>,
+					<44 0 &intc GIC_SPI 98  IRQ_TYPE_LEVEL_HIGH>,
 					<47 0 &intc GIC_SPI 93  IRQ_TYPE_LEVEL_HIGH>,
 					<48 0 &intc GIC_SPI 138 IRQ_TYPE_LEVEL_HIGH>,
 					<50 0 &intc GIC_SPI 139 IRQ_TYPE_LEVEL_HIGH>,
